pkgs/development/python-modules/web-cache/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, buildPythonPackage, fetchPypi, isPy3k }:
+
+buildPythonPackage rec {
+  pname = "web-cache";
+  version = "1.1.0";
+  disabled = !isPy3k;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1d8f1s3i0s3h1jqvjq6cp639hhbbpxvyq7cf9dwzrvvvr0s0m8fm";
+  };
+
+  # web_cache doesn't have tests
```

Comment:
The upstream repo contains [tests](https://github.com/desbma/web_cache/tree/master/tests).

Reply:
Whoops, should have made it clearer that the archive `fetchPypi` fetches doesn't come with tests.